### DISCUSSION TITLE
Fetch AWS S3 images

### DISCUSSION THREAD

**Discussion by JulcarlSelma** [vladmandic/face-api#117] [Q&A] [answered]
Hi, I Need help, I am new to face recognition and there this code below where actually works, but when I try to use AWS S3 links in fetchImage it shows Cors Policy issue. 

I know you can't set Access-Control-Allow-Origin header in client but is there any way to resolve this without setting the origin?

![image](https://user-images.githubusercontent.com/20558190/185838701-34ce6492-d4ad-41b4-87e3-5d6489017c4e.png)

**Comment by vladmandic** [comment, marked as answer]
forget about `face-api` for a second, can you use standard `fetch` to download image without it?  
if yes, you can get `face-api` to work as well. if no, you need to get fetch to work.

afaik, you cant bypass it by changing headers client-side , you need to configure aws s3 bucket to allow cors.

**Reply by JulcarlSelma** [reply]
I think you are right. I still can fetch it using the standard fetch. Thanks
